src/Utils/request/types.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Avoid using eslint-disable for an entire file, as it can unintentionally bypass important linting rules and introduce issues. Instead, disable ESLint only for specific lines or blocks of code where necessary, ensuring minimal impact.